docs/howto/manage-charms.rst

```diff
@@ -235,11 +235,12 @@ Add docs
 ~~~~~~~~
 
 If you publish your charm on Charmhub, reference documentation about the charm's
-resources, actions, configurations, relations, and libraries is extracted automatically.
-However, you should also aim to add further docs, e.g., a tutorial, how-to guides, etc.
-To provide a link to these docs, in your charm's project file, specify a
-value for the :ref:`links.documentation <charmcraft-yaml-key-documentation>` key. Note
-that at present this must be a Discourse page. E.g.,
+resources, actions, configurations, relations, and libraries is generated and 
+published automatically in respective tabs.
+
+To add content to the Description tab, 
```

Comment:
Any chance we can use bold to fit with the recommended UI formatting? https://docs.ubuntu.com/styleguide/en/#bold-vs.-italics